numjuggler/main.py

```diff
@@ -77,7 +86,7 @@ def tr2str(pl, fmt1='{:12.9f}', fmte='{:16.8e}'):
          'annotate', 'getc', 'mnew', 'combinec', 'cdens')
 
 
-def main():
+def main(args=sys.argv[1:]):
```

Comment:
Why the optional argument ``args`` is needed here? It is later passed to the ``parse_known_args()`` method that refers anyway to ``sys.argv`` (according to [docs](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_args)).

Reply:
To facilitate tests with the main() function.

Reply:
Ok, thanks.